### PR TITLE
748: judge note bug

### PR DESCRIPTION
**Pull request by kkoskelin**
https://trello.com/c/fbOulrss/748-cannot-create-judges-notes-on-case-with-docket-number-having-a-suffix

Screen Shot 2020-08-25 at 12 04 48 PM

### CODE REVIEW

File: web-client/src/views/TrialSessionWorkingCopy/AddEditUserCaseNoteModal.jsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const AddEditUserCaseNoteModal = connect(
         onConfirmSequence={onConfirmSequence}
       >
         <h5 className="margin-bottom-4">
-          Docket {modal.docketNumber}: {modal.caseTitle}
+          Docket {modal.docketNumberWithSuffix}: {modal.caseTitle}
```
Comment (matthopson) [inline web-client/src/views/TrialSessionWorkingCopy/AddEditUserCaseNoteModal.jsx:31]: Why not put this on the helper?

File: web-client/src/presenter/actions/TrialSessionWorkingCopy/setAddEditUserCaseNoteModalStateFromDetailAction.js [hunk 2]
```diff
@@ -20,8 +20,9 @@ export const setAddEditUserCaseNoteModalStateFromDetailAction = ({
   );
   const caseTitle = applicationContext.getCaseTitle(caseCaption || '');
 
+  store.set(state.modal.docketNumber, docketNumber);
   store.set(
-    state.modal.docketNumber,
+    state.modal.docketNumberWithSuffix,
     `${docketNumber}${docketNumberSuffix ? docketNumberSuffix : ''}`,
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/TrialSessionWorkingCopy/setAddEditUserCaseNoteModalStateFromDetailAction.js]: `docketNumberWithSuffix` should already be set on `caseDetail`, so it should not need to be generated here. I think this was just missed in a refactor.

Comment (matthopson) [reply web-client/src/presenter/actions/TrialSessionWorkingCopy/setAddEditUserCaseNoteModalStateFromDetailAction.js]: +1

**Comment by kkoskelin** [comment]
I've revised this effort, removing the helper entirely and moving the work entirely to the action belonging to the sequence which opens this modal.

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**